### DISCUSSION TITLE
The stats section on the Daisy UI homepage is animated - is this via Daisy UI?

### DISCUSSION THREAD

**Discussion by austriker27** [saadeghi/daisyui#714] [Q&A] [answered]
I really like this animation on the daisy UI homepage.

![Kapture 2022-04-13 at 13 46 43](https://user-images.githubusercontent.com/14855925/163267775-3d221248-79fd-44de-8235-3fa4c146db15.gif)

I dont see anything in the [stats component](https://daisyui.com/components/stat/) about animation - how is it done? is it done via daisy?

Thanks!

**Comment by saadeghi** [comment, marked as answer]
That's a Svelte component: https://github.com/saadeghi/svelte-countup
